### ISSUE TITLE
Icones pour helthcare=laboratory et autres helthcare=*

### ISSUE THREAD

**Issue by Bibi56** [cquest/osmfr-cartocss#76]
Pour aller au delà de #72 et pour préparer le projet du mois de mai :
Utilisation d'icônes ou de caractères de la police [fontawesome](https://fontawesome.com/how-to-use/on-the-web/setup/hosting-font-awesome-yourself), [icônes médicaux](https://www.w3schools.com/icons/fontawesome5_icons_medical.asp).

tag | icône | commentaire | code | unicode
-- |-- | ---- | ---- | --
helthcare=laboratory  | | une pipette, celle d'iD | fas fa-vial | [&#xf492;](https://www.w3schools.com/icons/tryit.asp?icon=fas_fa-vial&unicon=f492)
Labo test covid  | | deux pipettes | fas fa-vials | [&#xf493;](https://www.w3schools.com/icons/tryit.asp?icon=fas_fa-vials&unicon=f493)
Vaccinodrome | | une seringue | fas fa-syringe | [&#xf48e;](https://www.w3schools.com/icons/tryit.asp?icon=fas_fa-syringe&unicon=f48e)

**Comment by deuzeffe** [comment]
@Bibi56 on me souffle dans l'oreillette (merci @corentinfuret ) qu'il y aurait incompatibilité de licence (CC-BY 4.0 vs CC0)

**Comment by Bibi56** [comment]
@deuzeffe, l'oreillette @corentinfuret a raison pour les icônes mais tort pour les polices.

Font Awesome Free License
-------------------------

Font Awesome Free is free, open source, and GPL friendly. You can use it for
commercial projects, open source projects, or really almost whatever you want.
Full Font Awesome Free license: https://fontawesome.com/license/free.

# Icons: CC BY 4.0 License (https://creativecommons.org/licenses/by/4.0/)
In the Font Awesome Free download, the CC BY 4.0 license applies to all icons
packaged as SVG and JS file types.

# Fonts: SIL OFL 1.1 License (https://scripts.sil.org/OFL)
In the Font Awesome Free download, the SIL OFL license applies to all icons
packaged as web and desktop font files.

(...)

# Attribution
Attribution is required by MIT, SIL OFL, and CC BY licenses. Downloaded Font
Awesome Free files already contain embedded comments with sufficient
attribution, so you shouldn't need to do anything additional when using these
files normally.

---


Donc un lien depuis le site donnant l'attribution des polices dont celle-ci me semble suffisant. Je ne suis pas juriste !
Sinon on peut toujours garder les idées (pour avoir une compatibilité avec iD et donc les ajouter dans [Maki](https://github.com/mapbox/maki) ou [Temaki](https://ideditor.github.io/temaki/) ou [The Noun Project](https://thenounproject.com/term/injection/)
En fait avec The Noun Project on trouve en CC... mais je suis tombé sur des CCBY : 
* La [seringue](https://thenounproject.com/search/?q=syringe&i=1542052) 
* [Le tube à essai](https://thenounproject.com/b_b_Lee/collection/bottles/?i=160075) : un peu différent 
* [Les tubes à essai]https://thenounproject.com/search/?q=vials) : un peu différent 

Alors Inkscape est ton ami ?

**Comment by corentinfuret** [comment]
@Bibi56 Oui, quand je parlais de CC-BY 4.0 c'était uniquement pour les icônes SVG, mais vu que c'est ce qui nous intéresse ici je n'ai pas détaillé plus. Pour Maki et Temaki je n'ai malheureusement rien vu d’intéressant. The Noun Project étant aussi du CC-BY c'est le même problème que fontawesome...

**Comment by deuzeffe** [comment]
@Bibi56 merci. À ma précédente lecture, je me suis demandée si la licence SIL OFL 1.1 License était compatible CC0/PD.
@corentinfuret : les 2 (CC-BY/CC0) semblent co-exister. En fonction des icônes :/
Je sens que je vais bidouiller celle que j'ai trouvé en DP :P

**Comment by Bibi56** [comment]
@deuzeffe "celle" prend un s au pluriel :-D.

**Comment by deuzeffe** [comment]
@Bibi56 nan, celle : https://publicdomainvectors.org/en/free-clipart/Vector-image-of-syringe-and-tubes/26700.html
